src/node/index.ts

```diff
@@ -424,6 +441,32 @@ export default function PluginInspect(options: Options = {}): Plugin {
     configResolved(_config) {
       config = _config
       config.plugins.forEach(hijackPlugin)
+
+      const _createResolver = config.createResolver
+      // @ts-expect-error mutate readonly
+      config.createResolver = function (this: any, ...args: any) {
```

Comment:
A lot of Vite's resolving flow is through `createResolver`, otherwise the data is quite bland. It's not possible to only measure the `resolve` plugin it's using as it's hidden internally. So compromising with a `vite:resolve (+alias)` plugin name for now.